openstudiocore/src/model/ElectricLoadCenterDistribution.cpp

```diff
@@ -479,8 +481,14 @@ namespace detail {
 
 
   // Storage Converter Object Name
-  //bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter);
-  //void ElectricLoadCenterDistribution_Impl::resetStorageConverter();
+  bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter) {
```

Comment:
Like I said I already coded a validityCheck method for the elcd, it's just a matter of doing the opposite of that, so neither complicated nor hard to do since I already coded the logic. Documenting would need a bit of work though

Envoyé de mon iPhone

> Le 30 juil. 2016 à 16:49, Dan Macumber <notifications@github.com> a écrit :
> 
> In openstudiocore/src/model/ElectricLoadCenterDistribution.cpp:
> 
> > @@ -479,8 +481,14 @@ namespace detail {
> >  
> >  
> >    // Storage Converter Object Name
> > -  //bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter);
> > -  //void ElectricLoadCenterDistribution_Impl::resetStorageConverter();
> > +  bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter) {
> Well the real right solution is to embed the storage converter (or any other true child object) into the ELCD. This isn't really possible in IDF but could be done in JSON.
> 
> Switching the bussType automatically is great but it is complicated. You have to decide which methods can change it and in which circumstances, then you'd have to document it all for users. Totally fine to skip this for now.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>